### ISSUE TITLE
Pyfa no longer starts on Mac.

### ISSUE THREAD

**Issue by GKFC** [pyfa-org/Pyfa#2565]
Hi,

I have Mojave 10.14.6 and Pyfa was working fine until 2.57.3 when it no longer runs.

Clicking the icon tries to open the app, but nothing happens, the icon appears in the taskbar then disappears after a few seconds.

Not sure what's going on.

Cheers,

G.

**Comment by DarkFenX** [comment]
Does 2.57.2 work? What is the last version which works fine for you?

**Comment by GKFC** [comment]
Hi,

I hadn't updated in a while so 2.55.0 was the previous version that was working fine.

Neither 2.57.2 nor 2.58.0 work currently for me.

Cheers,

G.

**Comment by DarkFenX** [comment]
Try those two:

- https://github.com/pyfa-org/Pyfa/releases/tag/v2.56.0
- https://github.com/pyfa-org/Pyfa/releases/tag/v2.57.0

I assume the first will work and the second won't.

**Comment by GKFC** [comment]
Correct, 2.57.0 tries to start up but just hangs.  If you need any further information please let me know.

**Comment by DarkFenX** [comment]
Actually I am clueless on how to debug this. I have access to two different macbooks and pyfa runs just fine on both.

Second release I linked had multiple libraries which power pyfa updated. Debugging which one is causing the issue is complicated, and to be honest, with my packed irl schedule I just won't have time for this (unless more people experience it and maybe someone has python dev experience, ie could assist me with debugging). Sorry for that

**Comment by GKFC** [comment]
I understand, your Macbooks have Mojave 10.14.6 on it?  I haven't updated my OS in a while.

If you do need any additional logging, or run a version that can give you them please let me know.  I guess I'll try and update Pyfa in future to see if it resolves... right now I can't use it unfortunately.

**Comment by TheBlackPlague** [comment]
I am running macOS Sequoia 15.0.0 with the M3 Air chip and getting the same issue. It crashes the first time on launch. The second time launches two icons on the task window, one being the application (the second icon to launch).

**Comment by cynricfu** [comment]
> I am running macOS Sequoia 15.0.0 with the M3 Air chip and getting the same issue. It crashes the first time on launch. The second time launches two icons on the task window, one being the application (the second icon to launch).
> 
> And it starts dummy processes. Deleting it now.

I had similar issues on my MacBook Pro (M1 Pro chip with macOS 15.2). But later I found that pyfa can successfully run if I wait patiently (1~2 mins).

**Comment by GKFC** [comment]
Yes I'm seeing this after I've done an OS update (finally).

Pyfa will open, shut, but after about 10 seconds launch itself correctly.

So this is "kind of" fixed for now.

**Comment by TheBlackPlague** [comment]
This is not an adequate fix.

**Comment by GKFC** [comment]
nps I'll reopen it then.
